scripts/player.gd

```diff
@@ -9,16 +13,31 @@ extends CharacterBody2D
 @export_range(0, 1000, 10, "suffix:px/s") var speed: float = 500.0:
 	set = _set_speed
 
+## How fast does your character accelerate?
+@export_range(0, 5000, 1000, "suffix:px/s²") var acceleration: float = 5000.0
+
 ## How high does your character jump? Note that the gravity will
 ## be influenced by the [member GameLogic.gravity].
 @export_range(-1000, 1000, 10, "suffix:px/s") var jump_velocity = -880.0
 
+## Can your character jump a second time while still in the air?
+@export var double_jump: bool = false
```

Comment:
Yes I love that these platformer settings are part of the "moddability" (if that's a word in English).

Reply:
[descriptivism](https://en.wikipedia.org/wiki/Linguistic_description) tells us that if it's a word people use and understand, then it's a word :)